apps/web/src/app/(pages)/(dashboard)/roles/page.tsx

```diff
@@ -106,6 +108,8 @@ export default function Roles({
           </div>
         </div>
       )}
+      {reviews.isSuccess && reviews.data.length === 0 && <NoResults />}
```

Comment:
beautiful

Reply:
ok so i lied, if we do this then the search bar isn't rendered and adding it to the above return statements duplicates like an extra 15 lines of code so I think it might be better to leave where they are?

Reply:
you're right i didn't realize that this was a page and that we have the `SearchFilter` component on it